group_vars/matrix_servers

```diff
@@ -3166,3 +3176,52 @@ matrix_user_creator_users_auto: |
 # /matrix-user-creator
 #
 ######################################################################
+
+
+######################################################################
+#
+# matrix-user-verification-service
+#
+######################################################################
+
+## FIXME: Needs to be updated when there is a proper release by upstream.
+matrix_user_verification_service_docker_image: "{{ matrix_user_verification_service_docker_image_name_prefix }}matrixdotorg/matrix-user-verification-service@sha256:d2aabc984dd69d258c91900c36928972d7aaef19d776caa3cd6a0fbc0e307270"
```

Comment:
Why not put these things in the role itself? Currently, the role has some version defined... which has no chance of working right now.. And then we patch it up here to make it work.

We'd better keep this in the role.

Reply:
We decided to overwrite this in group_vars because the matrix-org/matrix-user-verification-service does not have any official releases after v2.0.0 (Feb 18, 2021) and we wanted to stick to a release tag in the role defaults. 
This does add some clutter to the group_vars and we should have clarified the comment such that it reads: 
```
# FIXME: When there is a new release for matrix-org/matrix-user-verification-service,
#   update matrix_user_verification_service_version in the role defaults and remove this.
```

We can also move the comment into the role defaults and use the working commit hash there.

In any case overwriting `matrix_user_verification_service_docker_image` is somewhat incorrect and we should only overwrite `matrix_user_verification_service_version`. I.e. `matrix_user_verification_service_version="sha256:d2aabc984dd69d258c91900c36928972d7aaef19d776caa3cd6a0fbc0e307270"`, which unfortunately does not work since docker pull requires an '@' instead of ':' as delimiter when using a digest. 

As a work around one could use `branch-master` as a version. We decided against this, since we did not want the container image to change without notice.